fiatconnect-api.md

```diff
@@ -1655,9 +1659,7 @@ represents what *kind* of account that schema represents; e.g., Debit Card, Cred
 
 ```
 [
-	`CheckingAccount`,
-	`DebitCard`,
-	`CreditCard`
+	`BankTransfer`
```

Comment:
`Transfer` sounds more like an operation than an account. Does this represent any bank account that support transfers? Would checking account / debit card all fall under `BankTransfer" or are they still separate types and if they are the same, what are other account types?

Also, do the examples above need to be updated?

Reply:
I'm not entirely sure. Checking account would probably fall under `BankTransfer`, but debit card might be a different account type entirely (e.g., `BankTransfer` could represent transfer methods that use ACH, but debit card payments are available in real time so could be considered another account type). It's hard to say exactly until we get more examples of required schemas/data from providers.

Reply:
How about just `BankAcount`?

Reply:
`BankAccount` sounds perfect to me.